includes/class-edd-html-elements.php

```diff
@@ -86,7 +86,7 @@ public function product_dropdown( $args = array() ) {
 				array(
 					'key'     => '_edd_product_type',
 					'value'   => 'bundle',
-					'compare' => 'NOT EXISTS',
+					'compare' => '!=',
```

Comment:
The `meta_query` needs to be updated to account both for products which do have the `_edd_product_type` meta key, and for products which no longer have the meta key. This appears to work:

```php
			$product_args['meta_query'] = array(
				'relation' => 'OR',
				array(
					'key'     => '_edd_product_type',
					'value'   => 'bundle',
					'compare' => '!=',
				),
				array(
					'key'     => '_edd_product_type',
					'value'   => 'bundle',
					'compare' => 'NOT EXISTS',
				),
			);
```

Reply:
@robincornett I believe since this was an unreleased bug and the meta can't exist on `2.9.x` we don't need to account for the times it was saved while testing `3.0`. 

@ashleyfae I know we have some 3.0 database migration routines that are required for those using pre-release 3.0 -- are we also trying to account for temporary data inaccuracies like this?

Reply:
@spencerfinnell Hm introduced in 2018 by the looks of it? That was quite a while ago...

I guess I'm okay with not accounting for it provided that re-saving the download would immediately fix the problem (which I'm guessing it would?).

My preference would be:

1. Don't account for it
2. Write an upgrade routine to fix it

Meta queries are already gross, so I'd rather not build one grosser than it has to be.

Reply:
@ashleyfae That meta query has been around since then but there hasn't been a possibility to save a value other than `bundle` since `3.0`.  The change by @LisaCee is to provide parity with the AJAX product search that has existed for a while too: https://github.com/easydigitaldownloads/easy-digital-downloads/issues/7811#issuecomment-621924359

Reply:
@spencerfinnell the PR changes the meta query check to `!=` instead of `NOT EXISTS`, so shouldn't that be reverted if we are not checking for both?

Reply:
@robincornett You're right -- we should update this instance and the other mentioned in https://github.com/easydigitaldownloads/easy-digital-downloads/issues/7811#issuecomment-621924359 to check if it is empty, or `!=` bundle. That will ensure Default (no meta) are returned, as well as potential non-`bundle` types (possibly added by extensions).

Sorry for the confusion ☕